DEV/scripts/ppr_binnenveld.R

```diff
@@ -9,5 +9,5 @@
   binnenveld <- fread('dev/data/binnenveld.csv')
 
   # save file
-  usethis::use_data(binnenveld)
+  usethis::use_data(binnenveld, version = 3, overwrite = TRUE, compress = 'xz')
   
```

Comment:
if needed you can make this set smaller. its only used for illustration purposes in the vignettes.

Reply:
The size of this dataset 37kb, which seems not a of big concern to me. The table waterstress.obic is 279kb. The arguments were added to make those settings explicit and to benefit from better serialization and compression